### ISSUE TITLE
Allowing any additional props

### ISSUE THREAD

**Issue by ryanatkn** [gcanti/tcomb-react#16]
I like the check for unwanted additional props, but is there a way to allow them? I'm currently dropping back to React PropTypes for higher order components that pass all props down to their children. I tried a few variations including `propTypes(t.union([t.Any, t.struct({...})]))` but haven't found a solution. Thanks for the great libraries!

**Comment by gcanti** [comment]
Hi @ryanatkn,

Checking unwanted additional props is a nice feature (when it's enabled I feel more confident of my code)
but since it's quite intrusive I think it should be possible for the user to opt-out. I'm planning to add an additional argument `options` to `propType()` to allow a finer configuration, something like:

```js
propTypes({
  name: t.Str,
  surname: t.maybe(t.Str)
}, { strict: false }) // opt-out: name and surname will still be checked, but additional props are allowed
```

I'd keep the default value of the `strict` option to `true`.

**Comment by gcanti** [comment]
Just published v0.8.8 with the new feature

**Comment by ryanatkn** [comment]
Fantastic, thank you!
